### ISSUE TITLE
Hyper-parameters for the classic implementation

### ISSUE THREAD

**Issue by WojciechMigda** [cair/fast-tsetlin-machine-with-mnist-demo#1]
Hi,
what hyper-parameters for the classic implementation would be needed to achieve the same level of accuracy with the same dataset (>98%)?
I assume the dataset used in the original paper is not the same as the one used for this experiment, is that correct?
Thank you.

**Comment by olegranmo** [comment]
Hi Wojciech! 
Thanks for your question. You can use exactly the same hyper-parameters (that is 1000 clauses per class). The two implementations only differ in computation speed and memory usage. 

The digits result from the paper is for a completely different and small-scale dataset (https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_digits.html). The dataset used to evaluate the new implementation is the full MNIST dataset (http://yann.lecun.com/exdb/mnist/). Hence, results are not comparable between the two.

Hope this clears things up! Otherwise, don't hesitate to get in touch.

BTW. Thanks for the solid Tsetlin Machine Toolkit you are building.
